src/main/resources/view/WidgetListViewBox.fxml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.ListView?>
+<?import javafx.scene.layout.StackPane?>
+<?import javafx.scene.layout.VBox?>
+
+<StackPane prefHeight="365.0" prefWidth="337.0" styleClass="widget-pane" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
+   <children>
+      <VBox fx:id="viewBox">
+         <children>
+            <StackPane prefHeight="30.0" prefWidth="337.0">
+               <children>
+                  <Label fx:id="header" alignment="CENTER" prefHeight="46.0" prefWidth="249.0" styleClass="widget_header"/>
+               </children>
+            </StackPane>
+            <ListView fx:id="countryNoteListView" VBox.vgrow="ALWAYS"/>
+         </children>
+      </VBox>
+   </children>
+</StackPane>
```

Comment:
What's this update?

Reply:
Ignore this, not sure why Github flags this as an update even thought nothing changed.